Makefile

```diff
@@ -0,0 +1,21 @@
+NPM_TOKEN ?= '00000000-0000-0000-0000-000000000000'
+CI_BUILD_NUMBER ?= $(USER)-snapshot
+VERSION ?= 0.2.$(CI_BUILD_NUMBER)
+
+test:
+	npm install --silent
+	npm test
+
+package:
+	docker run \
```

Comment:
I thought I had left a reply to this, but looks like it didn't stick.

With the way this was set up, there was no need for a Dockerfile or `docker pull` - the `docker run` command took care of all that (I think the `docker pull` was unnecessary because `node:6` is in the Docker registry so it was pulled automatically from there).

However, I've gone a completely different route with this that makes it a lot simpler by just using Travis's [built-in support](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) for Node-based projects that just need to run `npm test` - no need for Docker at all, although I have kept the Makefile just for the npm versioning helper. I'd like to avoid the complication of Docker for this repo in part because it's not "runnable", so containerization is a little overkill for running tests - I'm much more comfortable with the new setup.